src/icons/delete.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+type DeleteIconProps = {
+  onClick: () => void;
+};
+
+const DeleteIcon: React.FC<DeleteIconProps> = ({ onClick }) => {
```

Comment:
Je pense que le nom du fichier devrait être le même que le composant et commencer par une majuscule étant donné que c'est un composant.

Reply:
Fixed in another PR